### PR TITLE
Swapped RijndaelEncryptionProvider encryption with Aes

### PR DESCRIPTION
**Pull request by Yantrio**
part of #2220

### CODE REVIEW

**Comment by khellang** [comment]
There is a legit failure here. Both `FormsAuthenticationConfigurationFixture` and `FormsAuthenticationFixture` doesn't compile after this...

**Comment by Yantrio** [comment]
for now I am just focusing on getting the Nancy project working, then moving on to the tests and other projects too. If these PRs can be for initial discussion, the rest of the work on the other projects can just be a cut + paste solution.

File: src/Nancy/project.json [hunk 1]
```diff
@@ -9,6 +9,32 @@
...
+            "System.Diagnostics.Tools": "4.0.1-beta-23516",
+            "System.Dynamic.Runtime": "4.0.11-beta-23516",
+            "System.IO.FileSystem": "4.0.1-beta-23516",
+            "System.Linq": "4.0.1-beta-23516",
+            "System.Net.NameResolution": "4.0.0-beta-23516",
+            "System.Net.Primitives": "4.0.11-beta-23516",
+            "System.Reflection.TypeExtensions": "4.0.1-beta-23409",
+            "System.Reflection.Extensions": "4.0.1-beta-23516",
+            "System.Runtime": "4.0.21-beta-23516",
+            "System.Runtime.Serialization.Primitives": "4.0.11-beta-23409",
```
Comment (jeremymeng) [inline src/Nancy/project.json]: There's a RC1 version `"System.Runtime.Serialization.Primitives": "4.1.0-beta-23516"`

File: src/Nancy/project.json [hunk 2]
```diff
@@ -9,6 +9,32 @@
...
+            "System.ComponentModel.TypeConverter": "4.0.1-beta-23516",
+            "System.Console": "4.0.0-beta-23516",
+            "System.Data.Common": "4.0.1-beta-23516",
+            "System.Diagnostics.Tools": "4.0.1-beta-23516",
+            "System.Dynamic.Runtime": "4.0.11-beta-23516",
+            "System.IO.FileSystem": "4.0.1-beta-23516",
+            "System.Linq": "4.0.1-beta-23516",
+            "System.Net.NameResolution": "4.0.0-beta-23516",
+            "System.Net.Primitives": "4.0.11-beta-23516",
+            "System.Reflection.TypeExtensions": "4.0.1-beta-23409",
```
Comment (jeremymeng) [inline src/Nancy/project.json]: RC1 version: `"System.Reflection.TypeExtensions": "4.1.0-beta-23516"`

**Comment by khellang** [comment]
:watch:

File: src/Nancy/project.json (hunk 2 above)
Comment (Yantrio) [inline src/Nancy/project.json]: these have been addressed in a seperate PR

**Comment by Yantrio** [comment]
fixed tests, need re-review.

**Comment by khellang** [comment]
A test failed on Travis, should we be worried?

> Nancy.Tests.Unit.Sessions.CookieBasedSessionsFixture.Should_return_blank_session_if_hmac_missing [FAIL]

**Comment by Yantrio** [comment]
screen shot 2016-02-10 at 15 13 35
that's strange, it seems to pass on my machine, can't seem to replicate it falling over.

**Comment by Yantrio** [comment]
What do you think we should do? @khellang ? I can't seem to ever get the test to fail locally, might be a 1 off issue.

**Comment by jchannon** [comment]
:watch:

**Comment by Yantrio** [comment]
:watch:

**Comment by khellang** [comment]
First of all, this can't be merged because of conflicts.
Second, the history is off.
Third, the failing test makes me nervous and hesitant to merge this without reproducing it on a Linux machine. It's likely some native crypto stuff messing up.

:watch:

**Comment by Yantrio** [comment]
Rebased, tests passing, history fixed. LETS DO THIS!

**Pull request merged**